documentation/api/io.strimzi.api.kafka.model.Rack.adoc

```diff
@@ -0,0 +1,72 @@
+Configures rack awareness to spread partition replicas across different racks.
+
+A _rack_ can represent an availability zone, data center, or an actual rack in your data center.
+By configuring a `rack` for a Kafka cluster, consumers can fetch data from the closest replica.
+This is useful for reducing the load on your network when a Kafka cluster spans multiple datacenters.
+
+To configure Kafka brokers for rack awareness specify:
+
+* A `topologyKey` value to match the label of the cluster node used by Kubernetes when scheduling Kafka broker pods to nodes
+* The `RackAwareReplicaSelector` implementation for the Kafka `ReplicaSelector` plugin
+
+If the Kubernetes cluster is running on a cloud provider platform, the label must represent the availability zone where the node is running.
+Usually, nodes are labeled with the `topology.kubernetes.io/zone` label (or `failure-domain.beta.kubernetes.io/zone` on older Kubernetes versions),
+which can be used as the `topologyKey` value.
+
+The `ReplicaSelector` plugin provides the logic that enables clients to consume from the nearest replica.
+Specify `RackAwareReplicaSelector` for the `replica.selector.class` to switch from the default implementation.
+The default implementation uses `LeaderSelector` to always select the leader replica for the client.
+By switching from the leader replica to the replica follower, there is some cost to latency.
+If required, you can also customize your own implementation.
```

Comment:
I would probably move this and the first mantion of the RackAwareReplicaSelector only behind this first example.

Reply:
**Before example:**
![image](https://user-images.githubusercontent.com/47596553/98565915-14929b80-22a6-11eb-939a-70f2f8309dcc.png)

**After example:**
![image](https://user-images.githubusercontent.com/47596553/98565851-017fcb80-22a6-11eb-9691-29f659898656.png)